services/pkgreview/package-rating.service.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  name: Joi.string().required(),
+  type: Joi.string().required(),
+  icon: Joi.string().required(),
+  starString: [Joi.string().optional(), Joi.allow(null)],
+  reviewsCount: Joi.number(),
+  rating: [Joi.number().optional(), Joi.allow(null)],
+}).required()
+
+module.exports = class PkgreviewRating extends BaseJsonService {
+  static get category() {
+    return 'rating'
+  }
+
+  static get route() {
+    return {
+      base: 'pkgreview/rating',
+      pattern: ':pkgManager/:pkgSlug+',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'pkgreview.dev Package Ratings',
+        namedParams: { pkgManager: 'npm', pkgSlug: 'react' },
+        staticPreview: {
+          label: 'pkgreview.dev',
+          message: '4/5 (125)',
+          color: '#4F78FE',
+        },
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'pkgreview.dev',
+      color: '#4F78FE',
+    }
+  }
+
+  static render({ type, name, rating, reviewsCount }) {
+    return {
```

Comment:
We have conventions for service classes around separation of concerns and which function implements which. Currently, the render function is doing a combination of extraction/transformation of the raw data from the API _and_ rendering of the badge which is not what we want. 

At a high level, there's 3 core things that happen from within the `handle` function when a badge is requested:
1. retrieve data from the upstream provider (responsibility of the `fetch` function)
2. transformation/extraction of the relevant data from the response (this is a `transform` action that's either handled within the `handle` function or commonly within a separate `transform` function)
3. rendering (`render` function).

The conversion of raw rating should be extracted from render function accordingly